server/graphql/invoice/src/mutations/customer_return/delete.rs

```diff
@@ -54,7 +54,7 @@ pub fn map_response(from: Result<String, ServiceError>) -> Result<DeleteResponse
 }
 
 #[derive(Interface)]
-#[graphql(name = "DeleteOutboundReturnErrorInterface")]
+#[graphql(name = "DeleteCustomerReturnErrorInterface")]
```

Comment:
Feeling kinda concerned that this says changing `outbound` to `customer` rather than inbound, but I think its just git getting confused about the file renames? Looks like it all irons out ok...

Reply:
Yah... I triple checked this one but it all looked fine... weird it did this tho